go.mod

```diff
@@ -8,9 +8,11 @@ require (
 	code.gitea.io/gitea v1.22.1
 	code.gitea.io/sdk/gitea v0.19.0
 	github.com/AlecAivazis/survey/v2 v2.3.7
+	github.com/antihax/optional v1.0.0
 	github.com/bradleyfalzon/ghinstallation/v2 v2.11.0
 	github.com/cloudevents/sdk-go/v2 v2.15.2
 	github.com/fvbommel/sortorder v1.1.0
+	github.com/gdasson/bitbucketv1go v1.0.0
```

Comment:
is this a new bibtubkcet library ? we are having two library to interact with bbv1, that's going to be a burden in the future, any chance to only use one (either one) ?

Reply:
@chmouel yes, then we need to use this new one everywhere.

Reply:
is the work hard to do ?